mmv1/templates/terraform/examples/cloudbuild_trigger_repo.tf.erb

```diff
@@ -0,0 +1,72 @@
+terraform {
```

Comment:
This block & the provider definition on L8 should be superfluous & can be removed. (This applies to all HCL files)